api/HTMLInputElement.json

```diff
@@ -1301,7 +1301,7 @@
             },
             "opera_android": {
               "version_added": null
-            },
+          true
             "safari": {
               "version_added": true
```

Comment:
Something went wrong here. There are other instances farther down. Please check throughout. 

Do you know about this: http://jsoneditoronline.org/#/

Reply:
it's odd that the linter didn't catch this 🤔

Reply:
Oh haha, how’d this happen?  I think this was a part of my mistakes when manually editing the JSON myself, oops!